src/web/styles.scss

```diff
@@ -10,6 +10,19 @@ body {
   font-size: 15px;
 }
 
+// The clickable parts of the boxes should change the cursor to a pointer
+tm-question-text-with-info {
+  cursor: pointer;
+}
+
+tm-view-photo-popover {
+  cursor: default;
+}
+
+.card-header {
+  cursor: pointer;
+}
+
```

Comment:
We normally add the styling of components to their specific SCSS files, not to the global file